ui/input/src/handlerUtil.ts

```diff
@@ -30,18 +30,12 @@ export function sanToUci(san: string, legalSans: SanToUci | undefined): Uci | un
 export function sanCandidates(san: string, legalSans: SanToUci): San[] {
   // replace '=' in promotion moves (#7326)
   const lowered = san.replace('=', '').toLowerCase();
-  return Object.keys(legalSans).filter(function (s) {
-    return s.toLowerCase().startsWith(lowered);
-  });
+  return Object.keys(legalSans).filter(s => s.toLowerCase().startsWith(lowered));
 }
 
 export function destsToUcis(dests: Dests): Uci[] {
   const ucis: string[] = [];
-  for (const [orig, d] of dests) {
-    d.forEach(function (dest) {
-      ucis.push(orig + dest);
-    });
-  }
+  for (const [orig, d] of dests) d.forEach(dest => ucis.push(orig + dest));
```

Comment:
(This is mainly from opinion), but I do think that expanding the for loop would be more readable:

```ts
for (const [orig, d] of dests) {
  for (const dest of d) {
    ucis.push(orig + dest);
  }
}
```

Reply:
*especially because deciding on one makes the combination of forEach and for loops less confusing.

Reply:
If you put these suggestions into a PR I will merge them